### DISCUSSION TITLE
Pin version of JS script

### DISCUSSION THREAD

**Discussion by vinayaugustine** [plausible/analytics#1883] [Anything Else]
We use cloud-hosted plausible on our site and want to start using [Subresource Integrity](https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity). I've pinned other scripts that we use to a specific version. Is that possible with the plausible JS script? I'd hate for it to silently break if <https://plausible.io/js/plausible.js> changes on us.

**Comment by metmarkosaric** [comment]
that's not possible unfortunately. would be nice if we could send notifications when a new version of the script is available.

there are two more threads on this topic with some more details:

https://github.com/plausible/analytics/discussions/1834

https://github.com/plausible/analytics/discussions/380

**Reply by vinayaugustine** [reply]
Just so I'm clear -- is the impossible part publishing SRI hashes or versioning the JS script? (we are interested in the latter, not the former).

**Reply by metmarkosaric** [reply]
neither of these are possible at the moment unfortunately.
